cmd/telegraf/cmd_plugins.go

```diff
@@ -0,0 +1,240 @@
+// Command handling for configuration "plugins" command
+package main
+
+import (
+	"fmt"
+	"io"
+	"sort"
+	"strings"
+
+	"github.com/influxdata/telegraf/plugins/aggregators"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/processors"
+	"github.com/influxdata/telegraf/plugins/secretstores"
+	"github.com/urfave/cli/v2"
+)
+
+func getInputNames() []byte {
+	inputNames := make([]string, 0, len(inputs.Inputs))
+	for i := range inputs.Inputs {
+		inputNames = append(inputNames, fmt.Sprintf("inputs.%s\n", i))
+	}
+	sort.Strings(inputNames)
+
+	return []byte(strings.Join(inputNames, ""))
```

Comment:
I think this is more intuitive
```suggestion
	inputNames := make([]string, 0, len(inputs.Inputs))
	for i := range inputs.Inputs {
		inputNames = append(inputNames, "inputs."+i)
	}
	sort.Strings(inputNames)

	return []byte(strings.Join(inputNames, "\n"))
```
Same for the other plugin types and deprecation functions...